fossunited/setup.py

```
@@ -163,7 +163,7 @@ def get_custom_fields():
                 "fieldname": "group_type",
                 "fieldtype": "Select",
                 "label": "Group Type",
-                "options": "Chapter Main\nEvent Participants\nCFP Proposers\nAccepted Proposers\nRejected Proposers\nOther",  # noqa: E501
+                "options": "Chapter Event Participants\nChapter CFP Proposers\nEvent Participants\nCFP Proposers\nAccepted Proposers\nRejected Proposers\nOther",  # noqa: E501
```

Ruff (0.14.0)

Unused `noqa` directive (non-enabled: `E501`)

Remove unused `noqa` directive

(RUF100)